drivers/platform/maxim/max32670/maxim_timer.c

```diff
@@ -0,0 +1,320 @@
+/***************************************************************************//**
+ *   @file   maxim_timer.c
+ *   @brief  Source file for Maxim timer platform driver.
+ *   @author Ciprian Regus (ciprian.regus@analog.com)
```

Comment:
please add the correct author

Reply:
Timer code is based on max32690 timer platform driver and verified w/ SDK files and testing w/ hardware. Same thing was done with max32670 uart_stdio platform driver so original author was not changed since file is technically almost similar, if not fully, w/ an already existing code.

Reply:
alright, fine by me as long as the author agrees.